packages/block-editor/src/hooks/style.js

```diff
@@ -323,28 +305,52 @@ const withElementsStyles = createHigherOrderComponent(
 			'link'
 		);
 
-		// The Elements API only supports link colors for now,
-		// hence the specific omission of `link` in the elements styles.
-		// This might need to be refactored or removed if the Elements API
-		// changes or `link` supports styles beyond `color`.
-		const elements = skipLinkColorSerialization
-			? omit( props.attributes.style?.elements, [ 'link' ] )
-			: props.attributes.style?.elements;
-
-		const styles = compileElementsStyles(
-			blockElementsContainerIdentifier,
-			elements
-		);
+		let styles = [];
```

Comment:
Nit: Might be worth memoing the result and also grouping them into a block.

Reply:
> Can you clarify what you mean by this?

It simply means that with `useMemo` we can group this logic inside a function (block) for readability's sake. 😅

Reply:
Ha, okay thanks. With all this talk of blocks I just wanted to make sure 😄